apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/ViewPoolCandidatePage.tsx

```diff
@@ -55,99 +71,435 @@ export const ViewPoolCandidate = ({
-      {snapshotUserPropertyExists && (
-        <div
-          data-h2-flex-item="base(1of1) p-tablet(content)"
-          data-h2-text-align="base(center) p-tablet(right)"
-          data-h2-padding="base(x1, 0, 0, 0)"
-        >
-          <ToggleGroup.Root
-            type="single"
-            color="primary.dark"
-            value={preferRichView ? "text" : "code"}
-            onValueChange={(value) => {
-              if (value) setPreferRichView(value === "text");
-            }}
-          >
-            <ToggleGroup.Item value="text">
-              {intl.formatMessage({
-                defaultMessage: "Text",
-                id: "Ude1JQ",
-                description: "Title for the application's profile snapshot.",
-              })}
-            </ToggleGroup.Item>
-            <ToggleGroup.Item value="code">
-              {intl.formatMessage({
-                defaultMessage: "Code",
-                id: "m0JFE/",
-                description: "Title for the application's profile snapshot.",
-              })}
-            </ToggleGroup.Item>
-          </ToggleGroup.Root>
-        </div>
-      )}
-    </div>
+    </TableOfContents.Section>
   );
 
   let mainContent: React.ReactNode;
-  if (snapshotUserPropertyExists && preferRichView) {
+  if (showRichSnapshot) {
+    const snapshotCandidate = parsedSnapshot?.poolCandidates
+      ?.filter(notEmpty)
+      .find(({ id }) => id === poolCandidate.id);
+    const categorizedEssentialSkills = categorizeSkill(
+      poolCandidate.poolAdvertisement?.essentialSkills,
+    );
+    const nonEmptyExperiences = parsedSnapshot.experiences?.filter(notEmpty);
     mainContent = (
-      <UserProfile
-        applicant={parsedSnapshot}
-        subTitle={subTitle}
-        sections={{
-          myStatus: { isVisible: false },
-          hiringPools: { isVisible: false },
-          about: { isVisible: true },
-          language: {
-            isVisible: true,
-          },
-          government: {
-            isVisible: true,
-          },
-          workLocation: {
-            isVisible: true,
-          },
-          workPreferences: {
-            isVisible: true,
-          },
-          employmentEquity: {
-            isVisible: true,
-          },
-          roleSalary: { isVisible: true },
-          skillsExperience: {
-            isVisible: true,
-          },
-        }}
-        isNavigationVisible={false}
-      />
+      <>
+        {subTitle}
+        <TableOfContents.Section id={sections.minExperience.id}>
+          <TableOfContents.Heading
+            as="h4"
+            size="h5"
+            data-h2-margin="base(x2 0 x.5 0)"
+          >
+            {sections.minExperience.title}
+          </TableOfContents.Heading>
+          <p data-h2-margin="base(x1, 0)">
+            {intl.formatMessage({
+              defaultMessage:
+                "Requirement selection: <strong>Minimum experience or education requirement (2 years of post-secondary)</strong>.",
```

Comment:
That is a good question, this matches the mockup but it makes sense that we specify which selection they made now that we store that. @Jerryescandon since we have the ability to know which selection the user made, should we update this to show that instead of both options here?

Reply:
This makes sense to me, I looked at what Josh did for the "Application review" and just reused the component.

### from "review application" page:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/57f9a950-78ba-4b62-acfc-48cd66924cad)

We dont specify there the selection, but we DO have two distinct options on the Education Requirements page:

### Education requirements page:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/10fdc66a-cde7-4144-8687-59847340cb27)

So theoretically we could instead show the selection like:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/41308dce-f652-48e0-8295-a4e51601998a)

Reply:
Thanks! Updated in [update education requirement option display](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6743/commits/0ba0a43810067e6f86bdd377a83daeb5c439f941)

Reply:
Nice!